kms/tpmkms/tpmkms.go

```diff
@@ -786,7 +790,7 @@ func (k *TPMKMS) storeCertificateChainToWindowsCertificateStore(req *apiv1.Store
 	}
 
 	if err := k.windowsCertificateManager.StoreCertificate(&apiv1.StoreCertificateRequest{
-		Name:        fmt.Sprintf("capi:sha1=%s;store-location=%s;store=%s;", fp, location, store),
+		Name:        fmt.Sprintf("capi:sha1=%s;store-location=%s;store=%s;skip-find-certificate-key=%s", fp, location, store, skipFindCertificatKey),
```

Comment:
Are `location` and `store` properly encoded to work on a URI?

Reply:
For existing and valid values they are, when they're checked in the CAPI implementation. But I can change the implementation to `uv.Set`, or something similar, if preferred

Reply:
Fixed in 4d88201cc7a12b915cb355995878ae77be457e37